data/tests/data-macro-explanation.js

```diff
@@ -0,0 +1,46 @@
+const dataMacros = /^(?:compat|css_ref|cssanimatedproperties|cssinfo|csssyntax|webextbrowsercompat)$/i;

```

Comment:
Please add a specification text to the top of the file:

<pre>
/*
 *  Title: Test that data macros have a note above them pointing to data sources.
 *
 *  Example 1: If you are using the e.g. the {{compat}} or {{cssinfo}} macros, there should 
 *  be a hidden note for contributors in edit-mode, so that they know where the data comes 
 *  from and how to change it. 
 * 
 *  Implementation notes: This test checks if there is a note for each macro 
 *  (by checking whether the sibling element before the element containing the macro
 *  has class="hidden") and whether that note contains a link (i.e. an &lt;a href="http..."&gt; 
 *  element; it doesn't check whether the link is valid).
 * 
 */
</pre>